### ISSUE TITLE
IDEA: Additional button to connect to our Discord community support

### ISSUE THREAD

**Issue by jen4ph** [Joystream/joystream-org#846]
<img width="859" alt="Screenshot 2024-05-01 at 4 09 19 PM" src="https://github.com/Joystream/joystream-org/assets/96877192/318f1184-1e43-4d57-8026-34821da28ea7">

**Background**:
During meetings and interactions within the DAO, I have discovered on at least two occasions that individuals have expressed interest in building an app with us. However, on Joystream.org, we only provide a link that directs users to the Atlas repository. I believe this is insufficient, as someone who might be interested and possibly understands the potential may not have a technical background. I think they should have a visible option to connect with someone to have some initial answers to the questions they might have at least.

**Suggestion**:
I suggest adding a button under the "Launch your own video app" button that directs users to the Joystream Discord server. The button could be labeled "Community Support for Inquiries" or something similar. I believe this addition will provide us with a better understanding of the demand, interest, and concerns people have regarding setting up their own app. Moreover, from the perspective of interested parties, having this button could be more welcoming, as it demonstrates our readiness to support them and answer their questions.
While this may not be a priority at the moment, I believe that dedicating some attention and effort to this matter will be beneficial in the long run.

Thank you.

**Comment by bedeho** [comment]
> I suggest adding a button under the "Launch your own video app" button that directs users to the Joystream Discord server.

I appreciate the issue, but I think this sort of direction to Discord will totally break. Most people will not be in the discord, so they will just be send to onboarding process which worst case even requires making a Discord account, but at the very least requires authenticating to get into our server, after which the normal onboarding flow happens, which does not capture this sort of user I believe, they will just be sent to one of many channels open.

I think it would be better to have a Notion page or something like that which tells the user what to do, or to link to a specific section on the README on the repo which is for helping people get started building apps.

**Comment by jen4ph** [comment]
Good point, thanks for the feedback. Perhaps, we'll work on this later.
